network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -151,3 +151,7 @@ <h2 class="tw-h3-heading">{% trans "Don’t see the product you’re looking for
 
   <script src="{% static "_js/bg-search.compiled.js" %}" async type="module"></script>
 {% endblock %}
+
+{% block background-parallax %}
+  {% include "fragments/buyersguide/shape_background.html" %}
+{% endblock %}
```

Comment:
Can you add some context why you need to add this block again? This template inherits from the `base.html` where you also add the background. Shouldn't that background be inherited too?

Reply:
> @fessehaye I also just noticed that the background is jumping when you switch filters on the PNI homepage.
> 
>  Screen.Recording.2022-10-24.at.13.49.13.mov

resolved